stream-payment-order/payment-order-core/src/main/java/com/backbase/stream/paymentorder/PaymentOrderTaskExecutor.java

```diff
@@ -35,6 +33,8 @@ public class PaymentOrderTaskExecutor implements StreamTaskExecutor<PaymentOrder
 
     private final PaymentOrdersApi paymentOrdersApi;
 
+    private final String BANK_REFERENCE_ID = "BANKREFERENCEID";
```

Comment:
I prefer not to use Strings in the code to minimize any spelling errors. Right now we are only using this variable once, but it could come again in the future. 
It probably would make sense to move this to a different class entirely to make it reusable everywhere, but this is the only string right now and I didn't want to create an entire class for 1 string.

Reply:
I guess I was confused by the pr description, you used the same link twice for the spec, now looking at https://developer.backbase.com/apis/specs/payment/payment-order-service-api/2.11.0/operations/PaymentOrders/updatePaymentOrder/ I can see that this value is used in the `fieldName` parameter, I guess renaming this variable from `BANK_REFERENCE_ID ` to `BANK_REFERENCE_ID_FIELD_NAME` would avoid misconceptions

Reply:
I see. I will update that! Thanks